Lib/test/test_readline.py

```diff
@@ -210,8 +210,10 @@ def display(substitution, matches, longest_match_length):
         self.assertIn(b"result " + expected + b"\r\n", output)
         self.assertIn(b"history " + expected + b"\r\n", output)
 
-    @unittest.skipIf(is_editline,
```

Comment:
I think we still need to skip the test if editline is available since it uses a different configuration format than readline.